packages/ace-linters/tests/common-converters.tests.ts

```diff
@@ -1,23 +1,26 @@
 import {expect} from "chai";
-import {Range as AceRange} from "ace-code/src/range";
 import {
     CompletionItemKind,
 } from "vscode-languageserver-protocol";
 import {CommonConverter} from "../type-converters/common-converters";
-
+import {Editor} from "ace-code/src/editor";
+import {MockRenderer} from "ace-code/src/test/mockrenderer";
 
 describe('General Converters', () => {
+    let editor = new Editor(new MockRenderer());
+    const Range = editor.getSelectionRange().constructor;
```

Comment:
in tests we could keep using ace-code without hacks.

Reply:
Yes, but it's nice to check, if those hacks would work and not break in product